Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.0
+// swift-tools-version:4.2
```

Comment:
Match to [RequestKit](https://github.com/nerdishbynature/RequestKit/blob/master/Package.swift#L1)